src/Engines/Native/nativeInterfaces.ts

```diff
@@ -27,6 +27,7 @@ export interface INativeEngine {
     createTexture(): WebGLTexture;
     loadTexture(texture: WebGLTexture, data: ArrayBufferView, generateMips: boolean, invertY: boolean, srgb: boolean, onSuccess: () => void, onError: () => void): void;
     loadRawTexture(texture: WebGLTexture, data: ArrayBufferView, width: number, height: number, format: number, generateMips: boolean, invertY: boolean): void;
+    loadRawTexture2DArray(texture: WebGLTexture, data: Nullable<ArrayBufferView>, width: number, height: number, depth: number, format: number, generateMipMaps: boolean, invertY: boolean, samplingMode: number, compression: Nullable<string>, textureType : number) : void;

```

Comment:
did you mean the name this function `loadRawTexture2DArray` and the implemented `createRawTexture2DArray` ?

Reply:
The native engine interface method is called loadRawTexture2DArray because it is only loading data to the object that is create on the JS side. The NativeEngine method is called createRawTexture2DArray because it needs to override the method from ThinEngine and in will actually create the texture object.